samcli/commands/package/package_context.py

```diff
@@ -102,6 +104,7 @@ def run(self):
             self.template_file,
             global_parameter_overrides=self._global_parameter_overrides,
         )
+        self._warn_preview_runtime(stacks)
```

Comment:
No this will work without `--beta-features` flag. `sam build` and `sam local` only requires base images, which is available now. But deploying such functions won't work until runtime is announced as GA. This is to inform if anyone tries to move forward.